source/style-guide/style/code-examples.txt

```diff
@@ -99,50 +91,59 @@ Create a VM Running a Docker Host
       To see how to connect Docker to this machine, run:
       docker-machine env test
 
-#. Run docker-machine ls again to see the VM that you created.
+#. Run the following command to see the VM that you created.
 
-   The output should look as follows:
+   .. io-code-block::
+      :copyable: true 
+         
+      .. input:: 
+         :language: sh
+         :linenos:
 
-   .. code-block:: sh
+         docker-machine ls
+
+      .. output::
+         :language: sh
+         :visible: true
 
-      NAME ACTIVE DRIVER STATE URL SWARM
-      test virtualbox Running tcp://192.168.99.101:237
+         NAME ACTIVE DRIVER STATE URL SWARM
+         test virtualbox Running tcp://192.168.99.101:237
 
 Run the Application
 -------------------
 
-#. Run a container from the image. The application code uses the
+1. Run a container from the image. The application code uses the
    environment variables that you defined to connect to the MongoDB
    container.
 
    .. code-block:: sh
 
-      $ docker run --detach \
-        --env MONGO_HOST=$MONGO_HOST \ env MONGO_PORT=$MONGO_PORT \ env
-        --MONGO_SSL=$MONGO_SSL \ env MONGO_DATABASE=$MONGO_DATABASE \ env
-        --MONGO_USER=$MONGO_USER \ env MONGO_PASSWORD=$MONGO_PASSWORD \ publish
-        --5000:5000 \
-        guestbook-mongo:1.0
+      docker run --detach \
+      --env MONGO_HOST=$MONGO_HOST \ env MONGO_PORT=$MONGO_PORT \ env
+      --MONGO_SSL=$MONGO_SSL \ env MONGO_DATABASE=$MONGO_DATABASE \ env
+      --MONGO_USER=$MONGO_USER \ env MONGO_PASSWORD=$MONGO_PASSWORD \ publish
+      --5000:5000 \
+      guestbook-mongo:1.0
```

Comment:
I'm not sure, but is the `guestbook-mongo:1.0` also output? If so, can we apply the same io-code-block formatting we did to the Troubleshotting section?